libfwupdplugin/fu-common-windows.c

```diff
@@ -38,3 +39,327 @@ fu_common_get_kernel_cmdline_impl(GError **error)
 {
 	return g_strdup("");
 }
+
+static const gchar *
+fu_common_convert_tzinfo_to_olson_timezone_id(const gchar *standard_name)
+{
+	if (g_strcmp0(standard_name, "AUS Central") == 0)
```

Comment:
With such a large number of 1:1 matches, would this be better done as a static struct that you walk instead?

Reply:
Good idea, modified.